dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.sessionStore-1.0.feature

```diff
@@ -0,0 +1,7 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.sessionStore-1.0
```

Comment:
Lets take advantage of the `MAJOR.MINOR.MICRO.QUALIFIER` syntax to make the feature names more descriptive, such as:
`sessionStore-1.0.0.Database` and `sessionStore-1.0.0.JCache`
(see the `jsfProvider-2.2.0.*` features for an example)